src/sfCommand.ts

```diff
@@ -526,3 +536,8 @@ export namespace SfCommand {
     context?: string;
   }
 }
+
+function removeEmpty(obj: Record<string, unknown>): Record<string, unknown> {
```

Comment:
non-blocking: are we sure we want to remove empty key/values from the error? it might make it hard for consumers to depend on the type returned?